doc/modules/model_evaluation.rst

```diff
@@ -1378,6 +1379,36 @@ function::
     for an example of mean squared error usage to
     evaluate gradient boosting regression.
 
+.. _mean_squared_log_error:
+
+Mean squared log error
+----------------------
+
+The :func:`mean_squared_log_error` function computes a risk metric corresponding
+to the expected value of the logarithmic squared (quadratic) error or loss.
+
+If :math:`\hat{y}_i` is the predicted value of the :math:`i`-th sample,
+and :math:`y_i` is the corresponding true value, then the mean squared log error
+(MSLE) estimated over :math:`n_{\text{samples}}` is defined as
+
+.. math::
+
+  \text{MSLE}(y, \hat{y}) = \frac{1}{n_\text{samples}} \sum_{i=0}^{n_\text{samples} - 1} (\log (1 + y_i) - \log (1 +
+  \hat{y}_i) )^2.
+
+Here is a small example of usage of the :func:`mean_squared_log_error`
```

Comment:
Kaggle's [note](https://www.kaggle.com/wiki/RootMeanSquaredLogarithmicError) that "RMSLE penalizes an under-predicted estimate greater than an over-predicted estimate" may be valuable here.